core/sys/windows/types.odin

```diff
@@ -297,6 +298,14 @@ REASON_CONTEXT :: struct {
 }
 PREASON_CONTEXT :: ^REASON_CONTEXT
 
+BLENDFUNCTION :: struct {
+  BlendOp: BYTE,
+  BlendFlags: BYTE,
+  SourceConstantAlpha: BYTE,
+  AlphaFormat: BYTE,
+}
```

Comment:
Make sure to use tabs in your source files. We use tabs for indentation and spaces for alignment. I would recommend installing .editorconfig plugin for whatever editor you're using so that your editor can configure itself in accordance with the styles the projects use.